### PR TITLE
Added Content for ML Community.

### PR DESCRIPTION
**Pull request by NobodySS07**
### Solves Issue #125 
<Do not delete this template>
Description: <provide a description> Removed Lorem Ipsum for Community of Machine Learning.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>

### CODE REVIEW

File: client/src/components/Community/CommunityList.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const list = [
         id: 4,
         linkName: "machine-learning",
         name: "Machine Learning",
-        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit ut aliquam, purus sit amet luctus venenatis, lectus magna fringilla urna, porttitor rhoncus dolor purus non enim praesent elementum facilisis leo, vel fringilla est ullamcorper eget nulla facilisi etiam dignissim diam quis"
+        description: "We at the ML community ,believe in helping one another,So one can fit in the Data Science and Machine Learning environment as smooth as a Linear Regression Model fits. As Mr Andrew Ng Said 'A Machine Learning Model is Data Hungry,The more it gets the better it becomes', We aspire to train ourselves to be the best. So come and dwell into the world of Artificial Intelligence by joining the community. "
```
Comment (EmperorYP7) [inline client/src/components/Community/CommunityList.js]: You may want to include only this line..
"We at the ML community, believe in helping one another, so one can fit in the Data Science and Machine Learning environment as smooth as a Linear Regression Model fits."

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
I see the text is overflowing from its div. This is a great line though, so try to add it [here](https://github.com/houseofgeeks/hg/blob/front-end/client/src/components/Community/Communities/MachineLearning.community.js). You can cut short the line in `CommunityList.js` so as to make it look good enough on the home page.

File: client/src/components/Community/CommunityList.js (hunk 1 above)
Comment (NobodySS07) [reply client/src/components/Community/CommunityList.js]: Done , Issue Resolved @EmperorYP7

**Review by tusharjain0022 — APPROVED** [review]
LGTM 💯

**Review by EmperorYP7 — APPROVED** [review]
LGTM

**Review by vishalvibes — COMMENTED** [review]
There are a few grammatical mistakes like putting comma after space and not leaving space after period.  Please run this through Grammarly once, before we merge this!

**Review by vishalvibes — CHANGES_REQUESTED** [review]
There are a few grammatical mistakes like putting comma after space and not leaving space after period. Please run this through Grammarly once, before we merge this!

**Comment by NobodySS07** [comment]
@Vishal19111999  I have made those changes, please Review

**Review by ankiiitraj — APPROVED** [review]

**Pull request merged**